enctests/testframework/encoders/ffmpeg_encoder.py

```diff
@@ -172,15 +173,17 @@ def get_source_path(self) -> Tuple[pathlib.Path, str]:
 
         return path, symbol
 
-    def get_output_filename(self, test_name: str) -> pathlib.Path:
+    def get_output_filename(self, test_name: str) -> (pathlib.Path, str):
```

Comment:
If we do: `from typing import Tuple` at the top we could do like the following:
```suggestion
    def get_output_filename(self, test_name: str) -> Tuple[pathlib.Path, str]:
```

Reply:
Looks like the import Tuple was already there.